.github/workflows/pull-request-checks.yaml

```diff
@@ -170,6 +170,40 @@ jobs:
           MERGE_BRANCH: ${{ github.ref }}
         run: ./.github/workflows/pull-request-check-cpplint.sh
 
+  # I just added this to check if my stuff is working on github actions
+  windows-msi-package:
+    runs-on: windows-2019
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Fetch dependencies
+        run: |
+          choco install winflexbison3
+      - uses: microsoft/setup-msbuild@v1.0.1
+      - name: Configure with cmake
+        run: |
+          New-Item -ItemType Directory -Path build
+          Set-Location build
+          cmake ..
+      - name: Build Release
+        run: |
+          Set-Location build
```

Comment:
Why is there a `cmake ..` in the line 190, and a `Set-Location` here again? Is there a specific reason? They look extraneous to me.

Reply:
I see there's a `cmake --build .` in the next step as well.

Do you think it makes sense to kill the `Set-Location` here and to `cmake --build build` for consistency with the other jobs as well (like in #5446)?